config/initializers/chronic.rb

```diff
@@ -0,0 +1 @@
+Chronic.time_class = Time.zone
```

Comment:
This is a good thing to do in your initializer. Do tests fail if this is removed?